src/DacpacTool/PropertyParser.cs

```diff
@@ -0,0 +1,160 @@
+﻿
```

Comment:
Please remove this useless newline.

Reply:
It seems this newline is still here?

Reply:
I don't see it on my side:
![image](https://user-images.githubusercontent.com/1955530/147759473-50b7e9af-16ea-4848-bfd0-f47c874b794b.png)

Reply:
Nevermind, I wasn't looking straight ;). Must be the end of the year.